custom/enikshay/ucr/reports/beneficiary_register.json

```diff
@@ -332,12 +321,17 @@
                 "transform": {
                     "type": "translation",
                     "translations": {
-                        "pcp": "MBBS",
-                        "pcc-chemist": "Chemist",
-                        "pac": "Ayush",
-                        "plc": "Lab",
-                        "ps-fieldstaff": "Field Staff",
-                        "ppia-do": "PPIA Admin"
+                        "type": "translation",
+                        "translations": {
```

Comment:
are the doubly nested translations on purpose?

Reply:
no, that was a mistake. Fixed that.